engine/gamesys/src/gamesys/scripts/script_resource_liveupdate.h

```diff
@@ -153,8 +153,11 @@ namespace dmLiveUpdate
 
     /*# register and store a live update zip file
      *
-     * Stores a zip file and uses it for live update content.
-     * The path is renamed and stored in the (internal) live update location
+     * Stores a zip file and uses it for live update content. The contents of the
+     * zip file will be verified against the manifest to ensure file integrity.
+     * It is possible to opt out of the resource verification using an option passed
+     * to this function.
+     * The path is renamed and stored in the (internal) live update location.
```

Comment:
The path isn't renamed?
The path that the user passes in is left as-is.

Reply:
I did not change this. That part is from the original code comment. Is it not correct?

Reply:
If you look at the code, you'll see that it opens a text file (`liveupdate.txt.tmp`) and we write the path (the path that the user passes in) to that text file.

The path in question (the zip file) is never moved.

Reply:
Updated comment